packages/2d/src/lib/components/Latex.ts

```diff
@@ -93,13 +106,16 @@ export class Latex extends SVGNode {
   @signal()
   public declare readonly tex: Signal<string[] | string, string[], this>;
 
+  private color: CanvasStyle;
+
   public constructor(props: LatexProps) {
     super({
       fontSize: 48,
       ...props,
       svg: '',
     });
     this.svg(this.latexSVG);
+    this.color = this.fill();
```

Comment:
This stores the initial value, and not a reference for the signal. Let's just use `this.fill` where this is used.